src/orchestrate.scm

```diff
@@ -209,4 +209,18 @@
 	))
 
 ; -------------------------------------------------------------
+; Say "Hello recog-id"
+; TODO: Add a request to chat psi-rules. That way actions of psi-rules will
+; be composable by the planner/action-orchestrator.
+(DefineLink
+	(DefinedPredicate "Greet recognized person")
```

Comment:
Even an AIML rule would be better than this. The AIML syntax for this would be
```
<template>What's up <get name="recog-id">?</template>
```
which creates a psi-rule with the action
```
   ;; action
   (ListLink
      (Word "What's")
      (Word "up")
      (ExecutionOutput
         (DefinedSchema "AIML-tag get")
         (ListLink
            (Concept "recog-id"))))
```
which runs
```
(cog-execute! (Get (State (Concept "recog-id") (Variable "$x"))))
```
to get the actual name of the person.  Now, I'm not saying that you should write XML for this; that's not needed nor advisible, but it is an OK placeholder until we can get sureal saying things, a better placeholder than this.